### PR TITLE
chore: update licences to dual mit and apache

### PR DESCRIPTION
**Pull request by vasco-santos**

### CODE REVIEW

**Review by hugomrdias — APPROVED** [review]

**Comment by ribasushi** [comment]
@vasco-santos in golang land we use a unified `LICENSE.md`, instead of the two license files as in this repo. The benefit of doing so is that github's own `License` link will go the the right both-license file.

Encourage you to just copy ours: https://github.com/ipfs/ipfs-repository-template/blob/main/LICENSE.md

Background ( do not copy that one - it has wrong `ipfs.org` links )
https://github.com/ipld/go-car/pull/59/files

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
+The contents of this repository are Copyright (c) corresponding authors and
+contributors, licensed under the `Permissive License Stack` meaning either of:
+
+- Apache-2.0 Software License: https://www.apache.org/licenses/LICENSE-2.0
+  ([...4tr2kfsq](https://gateway.ipfs.io/ipfs/bafkreiankqxazcae4onkp436wag2lj3ccso4nawxqkkfckd6cg4tr2kfsq))
+
+- MIT Software License: https://opensource.org/licenses/MIT
+  ([...vljevcba](https://gateway.ipfs.io/ipfs/bafkreiepofszg4gfe2gzuhojmksgemsub2h4uy2gewdnr35kswvljevcba))
```
Comment (ribasushi) [inline LICENSE.md:8]: Whooops you copied the wrong one ;) 
We want the `dweb.link` ones, like here: https://raw.githubusercontent.com/ipfs/ipfs-repository-template/main/LICENSE.md

Comment (vasco-santos) [reply LICENSE.md]: ah, I got confused by the suggestion and thought it was the opposite 😆

**Review by ribasushi — APPROVED** [review]
Perfect!

File: packages/api/package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "db-types": "./scripts/cli.js db-types"
   },
   "author": "Hugo Dias <hugomrdias@gmail.com> (hugodias.me)",
-  "license": "MIT",
+  "license": "(Apache-2.0 AND MIT)",
```
Comment (ribasushi) [inline packages/api/package.json:19]: @vasco-santos actually one last question ( sorry for being such a PITA!!! )

Is this field supposed to be a SPDX identifier? If yes it should read ( like what you just copied ) `Apache-2.0 OR MIT`

Comment (vasco-santos) [reply packages/api/package.json:19]: Yes, you are right! https://docs.npmjs.com/cli/v8/configuring-npm/package-json#license

We have AND in all the repos in nft and web3, but it should be as you suggest

**Review by ribasushi — APPROVED** [review]
@vasco-santos I tried really hard to find more problems with this PR
I am sorry.
I failed you.

:shipit:

**Pull request merged**